lib/ecto/multi.ex

```diff
@@ -458,6 +458,37 @@ defmodule Ecto.Multi do
     run(multi, name, operation_fun({:all, fn _ -> queryable end}, opts))
   end
 
+  @doc """
+  Checks if there exists an entry matching the given query and stores a boolean in the multi.
+
+  Accepts the same arguments and options as `c:Ecto.Repo.exists?/2`.
+
+  ## Example
+
+      Ecto.Multi.new()
+      |> Ecto.Multi.exists?(:post, Post)
+      |> MyApp.Repo.transaction()
+
+      Ecto.Multi.new()
+      |> Ecto.Multi.exists?(:post, fn _changes -> Post end)
+      |> MyApp.Repo.transaction()
+  """
+  @spec exists?(
+          t,
+          name,
+          queryable :: Ecto.Queryable.t | (any -> Ecto.Queryable.t),
```

Comment:
The function should accept the `changes` type. There is a type declared for this in Ecto.Multi: `fun(result_type)`

Reply:
Thanks I didn't notice that. I also fixed it for `one` and `all`